src/chat-handler.ts

```diff
@@ -62,6 +62,11 @@ export class ChatHandler extends AbstractChatModel {
     });
   }
 
+  clearChat(): void {
```

Comment:
The `AbstractChatModel` provides a `clearMessages()` method, which should delete all the message and trigger a signal. We should probably keep its name and call it here.